microcosm_pubsub/result.py

```diff
@@ -153,6 +153,18 @@ def log(self, logger, opaque):
             },
         )
 
+    def error_reporting(self, sentry_enabled, opaque):
+        opaque = opaque.as_dict()
+        if not sentry_enabled:
+            return
+        from sentry_sdk import capture_exception
+        from sentry_sdk import configure_scope
+        with configure_scope() as scope:
+            scope.set_tag("x-request-id", opaque.get("X-Request-Id"))
+            scope.set_tag("message-id", opaque.get("message_id"))
+            scope.set_tag("media-type", opaque.get("media_type"))
```

Comment:
ideally, we'd set scope when we *start* handing as message, so these tags would be available for (eg) error-level logs. but not for this PR.